client/src/containers/AddressPage.tsx

```diff
@@ -38,6 +38,7 @@ type RouteState = {
 type AddressPageProps = RouteComponentProps<RouteParams, {}, RouteState> &
   withMachineProps & {
     currentTab: number;
+    requestNewPortfolioData?: boolean;
```

Comment:
When I see `requestNewPortfolioData` my first thought is that it is used to clear some kind of cache, e.g. "don't use the old portfolio data you might have cached, request it anew so our data is fully up to date".  It took looking at the implementation to realize that it's about requesting the new wowza/graph structured format of portfolio data, rather than the legacy algorithm's.

I wonder if this should be called `requestPortfolioGraphData`, or `requestWowzaPortfolioData`, or `useNewPortfolioMethod` (as it is some other places in the code) or something else, to make it less ambiguous?

Reply:
yes! I like `useNewPortfolioMethod`